src/freenas/etc/rc.conf.local

```diff
@@ -201,7 +200,7 @@ _interface_config()
 			else
 				int_passopt=""
 			fi
-			if [ "${failover_status}" = "MASTER" ]; then
+		if [ ${failover_licensed} -eq 1 ]; then
```

Comment:
I meant to keep using failover_status, we dont want carp1 in both nodes to have skew 1